src/pyBiodatafuse/constants.py

```diff
@@ -96,7 +96,7 @@
     "el": float,
 }
 DISGENET_DISEASE_COL = f"{DISGENET}_diseases"
-
+OPENTARGETS_DISEASE_COL = "OpenTargets_diseases"
```

Comment:
do you use this variable?
it is not currently used in the workflow
if you want it change it to
```
OPENTARGETS_DISEASE_COL = f"{OPENTARGETS}_diseases"
```